cmd/storage-rest-client.go

```diff
@@ -267,7 +270,9 @@ func (client *storageRESTClient) ListVols(ctx context.Context) (vols []VolInfo,
 	}
 	defer http.DrainBody(respBody)
 	vinfos := VolsInfo(vols)
-	err = msgp.Decode(respBody, &vinfos)
+	dec := msgpNewReader(respBody)
+	defer readMsgpReaderPool.Put(dec)
+	err = vinfos.DecodeMsg(dec)
```

Comment:
This is already done internally, so no need for this.
```suggestion
	err = msgp.Decode(respBody, &vinfos)
```